hosting/src/views/Home/Home.html

```diff
@@ -114,6 +116,16 @@
         you have an idea for an event but need help organising then we also
         welcome your submissions.
       </p>
+      <p
+        class="lg:max-w-xl text-gray-500 text-base sm:text-lg md:text-xl mt-3 sm:mt-5 md:mt-5"
+      >
+        Phits Tech มุ่งหวังที่จะรวบรวมชุมชนเทคโนโลยีทั้งในและรอบๆพิษณุโลก
```

Comment:
At the moment the Thai text is after the English text. I think we should be a Thai-first community! **Please move the Thai text above the English text.**